flag_test.go

```diff
@@ -426,7 +426,11 @@ func TestStringFlagHelpOutput(t *testing.T) {
 		fl := &StringFlag{Name: test.name, Aliases: test.aliases, Usage: test.usage, Value: test.value}
 		// create a tmp flagset
 		tfs := flag.NewFlagSet("test", 0)
-		fl.Apply(tfs)
+		err := fl.Apply(tfs)
+		if err != nil {
+			t.Error(err)
+			return
+		}
```

Comment:
I'm happy to start using `stretchr/testify` more around here, e.g.:

```suggestion
		require.Nil(t, fl.Apply(tfs))
```

Reply:
Do we want to introduce another import in go.mod?  From my perspective it seems that this package has very few external imports.  Maybe in another PR as there are many use cases for using testify.

Reply:
https://github.com/urfave/cli/issues/1695